### PR TITLE
Create functionality

### PR DESCRIPTION
**Pull request by ArtyG-10**

### CODE REVIEW

File: app/javascript/components/pages/Portfolio.js [hunk 1]
```diff
@@ -42,7 +98,7 @@ class Portfolio extends Component {
                 </Row>
               );
             })}
-          {console.log(this.props.portfolio && this.props.portfolio)}
+          {console.log(this.props.portfolios && this.props.portfolios)}
```
Comment (sjproctor) [inline app/javascript/components/pages/Portfolio.js:101]: What do we have going on here?

File: app/javascript/components/pages/Portfolio.js [hunk 2]
```diff
@@ -1,11 +1,18 @@
 import React, { Component } from "react";
-import { Card, Row, Col, CardText, CardTitle } from "reactstrap";
-import { Link } from "react-router-dom";
+import { Card, Row, Col, CardText, CardTitle, Dropdown, DropdownToggle, DropdownMenu, DropdownItem, Button, Modal, ModalHeader, ModalBody, ModalFooter, Form, FormGroup, Input, Label} from "reactstrap";
+// import { Modal, Button } from "react-bootstrap";
```
Comment (sjproctor) [inline app/javascript/components/pages/Portfolio.js:3]: Looks like you already handled this import.

File: app/javascript/components/pages/NewPortfolio.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React, { Component} from 'react';
+
```
Comment (sjproctor) [inline app/javascript/components/pages/NewPortfolio.js:2]: What is the goal for this component?

File: app/javascript/components/pages/Portfolio.js (hunk 1 above)
Comment (ArtyG-10) [reply app/javascript/components/pages/Portfolio.js]: deleted this console log

File: app/javascript/components/pages/Portfolio.js (hunk 2 above)
Comment (ArtyG-10) [reply app/javascript/components/pages/Portfolio.js]: thanks removed that line

File: app/javascript/components/pages/NewPortfolio.js (hunk 3 above)
Comment (ArtyG-10) [reply app/javascript/components/pages/NewPortfolio.js]: didn't end up needing it- we removed it

**Review by sjproctor — APPROVED** [review]
Great!

**Pull request merged**